test/parallel/test-zlib-close-in-ondata.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const common = require('../common');
+const zlib = require('zlib');
+
+const ts = zlib.createGzip();
+const buf = Buffer.allocUnsafe(1024 * 1024 * 20);
```

Comment:
Can we use a fixed piece of data here? This might make the test a bit more memory-intensive than it needs to be, and using `allocUnsafe()` means that the compression work that ends up being done can vary widely, with no possibility of reproducing errors if they occur

Reply:
Fixed